mis_builder_report_xlsx/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "MIS Builder Report Export Xlsx",
+    "summary": """
+        
+    """,
+    "author": "Solvos",
+    "license": "AGPL-3",
+    "version": "15.0.1.0.0",
+    "category": "Project",
+    "website": "https://github.com/solvosci/slv-mis-builder",
+    "depends": [

```

Comment:
I miss some extra addon dependencies, please check `report_xlsx.py` code and models and fields used there, or try installing it in a new database and execute it, errors should be raise.

Reply:
The original repository is https://github.com/OCA/reporting-engine/blob/a2240527a2f65f578970fcb5d57e5969a622af9d/requirements.txt#L5C1-L5C9 But I'll add it anyway in a specific version

Reply:
OK for the python dependency.

Regarding to addons dependencies, it's still incomplete, we'll take a look together